presto-common/src/main/java/com/facebook/presto/common/plan/PlanCanonicalizationStrategy.java

```diff
@@ -47,6 +47,12 @@
      * This is used in context of history based optimizations.
      */
     CONNECTOR,
+
```

Comment:
Move this above CONNECTOR. And comment like it

EXACT strategy will canonicalize plan according to DEFAULT strategy, but supports all plan nodes.(see connector comments)

For CONNECTOR, you can modify comment:
CONNECTOR strategy will canonicalize plan according to EXACT strategy, and additionally...